.travis.yml

```diff
@@ -1,10 +1,6 @@
-sudo: required
-
 language: minimal
 
-env:
-    global:
-        - COMPOSE_PROJECT_NAME: sampleconfiguration
+dist: bionic
```

Comment:
Should roughly match https://github.com/libero/infrastructure/blob/1cc0fd2f5db4bc1f0a4b52dfc78728e7b1c98e9a/tf/main.tf#L3 now.

Reply:
there is a [pinned version](https://github.com/libero/infrastructure/blob/1cc0fd2f5db4bc1f0a4b52dfc78728e7b1c98e9a/tf/install-docker.sh#L23) there

Reply:
```
$ docker-compose version
docker-compose version 1.23.1, build b02f1306
docker-py version: 3.5.0
CPython version: 3.6.7
OpenSSL version: OpenSSL 1.1.0f  25 May 2017
```
We could upgrade to 1.23.1 in `infrastructure`, but want to check what is being used in all other builds first.

Reply:
[before the change](https://travis-ci.com/libero/sample-configuration/builds/117814813):
```
0.84s$ docker-compose version
docker-compose version 1.17.1, build 6d101fb
docker-py version: 2.5.1
CPython version: 2.7.13
OpenSSL version: OpenSSL 1.0.1t  3 May 2016
```